digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/MunicipalityApi.scala

```diff
@@ -966,7 +1075,25 @@ class MunicipalityApi(val onOffLinearAssetService: OnOffLinearAssetService,
     }
   }
 
-  delete("/:assetTypeName/:assetId"){
+  //Description of Api entry point to delete assets according asset type name and Id
+  val deleteAssetOnMunicipalityById =
+    (apiOperation[Long]("deleteAssetOnMunicipalityById")
+      .parameters(
+        pathParam[String]("assetId").description("Id of the asset to be deleted"),
+        pathParam[String]("assetTypeName").description("Asset type name to delete")
```

Comment:
"Asset type name to delete" maybe could be change to "Asset type name", because we will not delete the asset type name